uint8 rACT 
uint8 rMOD 
uint8 rGTO 
uint8 rATR 
uint8 rGLV 
uint8 rICF 
uint8 rICS 
uint8 rPRA 
uint8 rSPA 
uint8 rFRA 
uint8 rPRB 
uint8 rSPB 
uint8 rFRB 
uint8 rPRC 
uint8 rSPC 
uint8 rFRC 
uint8 rPRS 
uint8 rSPS 
uint8 rFRS
